packages/tldraw/src/lib/ui/hooks/useUnlockedSelectedShapesCount.ts

```diff
@@ -0,0 +1,34 @@
+import { useEditor, useValue } from '@tldraw/editor'
+
+/**
+ * Returns true if the number of UNLOCKED selected shapes is at least min or at most max.
+ */
+export function useUnlockedSelectedShapesCount(min?: number, max?: number) {
```

Comment:
this name is throwing me off a bit - should it be more like `useIsUnlockedSelectedShapesClamped` since it's a boolean, not a number being returned?